docs/user-guide/resource-limits.rst

```diff
@@ -0,0 +1,71 @@
+=========================
+Container resource limits
+=========================
+
+*Note: preliminary summary from skimming docs and educational guessing.*
+*No evaluation done. It could contain errors.*
```

Comment:
Should this be a [sphinx note](http://www.sphinx-doc.org/en/1.4.8/markup/para.html#directive-note)? :)